NEWS.md

```diff
@@ -1,6 +1,15 @@
 MathOptInterface (MOI) release notes
 ====================================
 
+v0.9.9 (December 29, 2019)
+---------------------
+
+- Added MathOptFormat.jl as FileFormats submodule (#969).
```

Comment:
This could be written to be more meaningful for casual readers. E.g., Incorporated MathOptFormat.jl as the FileFormats submodule. FileFormats provides readers and writers for a number of standard file formats and MOF, a file format specialized for MOI.